types/preview/@ember/array/-private/native-array.d.ts

```diff
@@ -4,10 +4,11 @@ declare module '@ember/array/-private/native-array' {
   import MutableArray from '@ember/array/mutable';
   import Mixin from '@ember/object/mixin';
 
-  // Get an alias to the global Array type to use in inner scope below.
+  // Get aliases to the global Array types to use in inner scope below.
   type Array<T> = T[];
+  type ReadonlyArray<T> = readonly T[];
```

Comment:
This actually just duplicates a built-in type of the same name, and we don't *need* to otherwise reference it past this.

```suggestion
```

Reply:
That's true... but the same is true for the `Array` type above. It's also built-in. I wasn't sure why it was added but decided to be consistent. I'm fine both ways. I'll remove the `ReadonlyArray` if you prefer?

Reply:
The `Array` type has historically been *shadowed* locally in a way that `ReadonlyArray` hasn't, and so we needed a way to refer to it distinctly. (Totally fair not to have that context.) Here, I suspect it could go away as well, but we should tackle that separately!